src/main/java/seedu/address/model/tag/Genre.java

```diff
@@ -12,8 +13,9 @@
 public class Genre extends Tag {
 
     public static final String MESSAGE_CONSTRAINTS = "Genre tags names should be a valid genre name";
-    public static final HashSet<String> VALID_GENRES = new HashSet<String>(Arrays.asList(
-            "pop", "rock", "jazz", "r&b", "classical", "hiphop", "country", "metal"
+    public static final HashSet<String> VALID_GENRES = new LinkedHashSet<>(Arrays.asList(
```

Comment:
Good that you incorporated the feedback from the recent peer review session to include more valid genres and instruments!